content/en/integrations/guide/pivotal-cloud-foundry-manual-setup.md

```diff
@@ -0,0 +1,502 @@
+---
+title: Pivotal Cloud Foundry Manual Setup Guide
+kind: guide
+description: "Steps for manually setting up the Pivotal Cloud Foundry Integration"
+aliases:
+  - /integrations/pivotal-platform
+further_reading:
+- link: "https://www.datadoghq.com/blog/monitor-tanzu-application-service/"
+  tag: "Blog"
+  text: "Monitor applications running on VMware Tanzu Application Service"
+---
+
+## Overview
+
+Pivotal Cloud Foundry (PCF) deployments can send metrics and events to Datadog. You can track the health and availability of all nodes in a deployment, monitor the jobs they run, collect metrics from the Loggregator Firehose, and more. Use this page to manually set up monitoring for your PCF application.
+
+There are three main components for the PCF integration with Datadog. First, the buildpack is used to collect custom metrics from your applications. Second, the BOSH Release collects metrics from the platform. Third, the Loggregator Firehose Nozzle collects all other metrics from your infrastructure. Read the [Datadog VMware Tanzu Application Service architecture][32] guide for more information.
```

Comment:
```suggestion
There are three main components for the PCF integration with Datadog. First, the buildpack is used to collect custom metrics from your applications. Second, the BOSH Release collects metrics from the platform. Third, the Loggregator Firehose Nozzle collects all other metrics from your infrastructure. For more information, see the [Datadog VMware Tanzu Application Service architecture guide][32].
```

Reply:
For this, I think it's better to use `read` instead of `see`, as it implies a more focused attention. Would you be open to modifying your suggestion to incorporate `read`?